ui/tests/integration/components/mfa/method-form-test.js

```diff
@@ -38,11 +39,11 @@ module('Integration | Component | mfa-method-form', function (hooks) {
       .exists(`Max validation attempts field input renders`);
   });
 
```

Comment:
let me take a look again at this one. I think I just need to return the id on the payload. Stay tuned.